server/index.js

```diff
@@ -22,3 +23,42 @@ app.use(parser.json());
 app.use('/test', (req, res) => {
   res.send('3-2-1 testing! Server is serving!');
 })
+
+// To render items in Bonus Tier (passed postman)
+app.get('/bonus/:bundleId', (req, res) => {
+  let bundleId = req.params.bundleId;
+
+  db.getBonuses(bundleId, (err, results) => {
+    if (err) {
+      res.send(err);
+    } else {
+      res.json(results);
+    }
+  })
+})
+
+// To render individual bonus upon click selection
+app.get('/bonus-title', (req, res) => {
+  let selectedTitle = req.query.title;
+
+  db.getBonusItem(selectTitle, (err, results) => {
```

Comment:
I think this should be `selectedTitle` as the first passed in argument here based on the declaration on line 42.

Reply:
Changed it to selectedTitle.